docs/src/main/asciidoc/maven-tooling.adoc

```diff
@@ -7,6 +7,14 @@ https://github.com/quarkusio/quarkus/tree/master/docs/src/main/asciidoc
 
 include::./attributes.adoc[]
 
+[[prerequisites]]
+== Prerequisites
+
+A working link:https://maven.apache.org/install.html[installation of Apache Maven] is required.
+
+IMPORTANT: Make sure that the environment variable `MAVEN_HOME` is pointing to your Maven installation. +
+Otherwise _none_ of the "global" settings from `conf/settings.xml` will be picked up by Quarkus.
```

Comment:
This is a little strongly worded and would scare users who don't really understand what that configuration usually does (which is pretty much nothing by default).
How about we word it a in way that lets users know that **if** they have changed any settings in `conf/settings.xml`, then either `MAVEN_HOME` or the `maven.home` system property needs to be set?

Reply:
Ok, I'll try.

If we start mentioning `maven.home` here we should also implement @aloubyansky's idea of generating a `systemPropertyVariables` entry, no?

Reply:
Yeah, that makes perfect sense

Reply:
So I'll take a stab at this. Once I'm done someone with a IntelliJ installation should check how it reacts to `${maven.home}` in `systemPropertyVariables`.

Reply:
Do we want to have this new `systemPropertyVariable` _everywhere_ where we already have the `LogManager` configured?
```
$ grep -Rl --exclude-dir=target '<java.util.logging.manager>org.jboss.logmanager.LogManager</java.util.logging.manager>' .
./build-parent/pom.xml
./core/builder/pom.xml
./devtools/aesh/dependency-reduced-pom.xml
./devtools/aesh/pom.xml
./devtools/platform-descriptor-json/src/main/resources/templates/basic-rest/java/pom.xml-template.ftl
./devtools/platform-descriptor-json/src/main/resources/templates/basic-rest/kotlin/pom.xml-template.ftl
./devtools/platform-descriptor-json/src/main/resources/templates/basic-rest/scala/pom.xml-template.ftl
./docs/src/main/asciidoc/building-my-first-extension.adoc
./docs/src/main/asciidoc/getting-started-testing.adoc
./docs/src/main/asciidoc/getting-started.adoc
./docs/src/main/asciidoc/logging.adoc
./docs/src/main/asciidoc/tests-with-coverage.adoc
./extensions/amazon-lambda/maven-archetype/src/main/resources/archetype-resources/pom.xml
./extensions/amazon-lambda-http/maven-archetype/src/main/resources/archetype-resources/pom.xml
./extensions/azure-functions-http/maven-archetype/src/main/resources/archetype-resources/pom.xml
./extensions/funqy/funqy-amazon-lambda/maven-archetype/src/main/resources/archetype-resources/pom.xml
./extensions/grpc/deployment/pom.xml
./independent-projects/tools/common/src/test/resources/templates/basic-rest/java/pom.xml-template.ftl
./independent-projects/tools/common/src/test/resources/templates/basic-rest/kotlin/pom.xml-template.ftl
./independent-projects/tools/common/src/test/resources/templates/basic-rest/scala/pom.xml-template.ftl
./integration-tests/bootstrap-config/application/pom.xml
./integration-tests/container-image/maven-invoker-way/src/it/container-build-docker/pom.xml
./integration-tests/container-image/maven-invoker-way/src/it/container-build-jib/pom.xml
./integration-tests/container-image/maven-invoker-way/src/it/container-build-multiple-tags-docker/pom.xml
./integration-tests/container-image/maven-invoker-way/src/it/container-build-multiple-tags-jib/pom.xml
./integration-tests/container-image/maven-invoker-way/src/it/container-image-push/pom.xml
./integration-tests/grpc-health/pom.xml
./integration-tests/grpc-interceptors/pom.xml
./integration-tests/grpc-mutual-auth/pom.xml
./integration-tests/hibernate-orm-panache/pom.xml
./integration-tests/kubernetes/maven-invoker-way/src/it/kubernetes-docker-build-and-deploy/pom.xml
./integration-tests/kubernetes/maven-invoker-way/src/it/kubernetes-jib-build-and-deploy/pom.xml
./integration-tests/kubernetes/maven-invoker-way/src/it/kubernetes-with-existing-manifest/pom.xml
./integration-tests/kubernetes/maven-invoker-way/src/it/openshift-s2i-build-and-deploy/pom.xml
./integration-tests/kubernetes/src/it/openshift-s2i-build-and-deploy/pom.xml
./integration-tests/maven/src/test/resources/projects/multimodule/runner/pom.xml
./integration-tests/maven/src/test/resources/projects/multimodule-revision-prop/runner/pom.xml
./integration-tests/maven/src/test/resources/projects/property-overrides/runner/pom.xml
./integration-tests/picocli/pom.xml
./integration-tests/spring-data-jpa/pom.xml
./integration-tests/vault/pom.xml
```
`platform-descriptor-json` is obvious but I am not sure about:
- _all_ docs
- Quarkus internal pom.xml files

Would be more consistent to do it for all, though.

Reply:
I'd say yes. `/independent-projects/tools/common/src/test/resources/templates/basic-rest/*` are formal templates, could be excluded.

Reply:
Phew, I had a quick look at some of those `pom.xml` (especially under `integration-tests`) and it all seems a bit inconsistent.
Why are so many `integration-tests`-projects redefining `maven-surefire-plugin`?
At least one of them is also passing the `LogManager` via `maven-failsafe-plugin`.

I'd say this should be cleaned up first in a separate PR. WDYT?

Reply:
It could surely be reorganized to not repeat common config. If you are up for it, sure :)

Reply:
A cleanup is certainly welcome if you want to take up the task 😉

Reply:
@geoand 

> This is a little strongly worded and would scare users who don't really understand what that configuration usually does (which is pretty much nothing by default).
> How about we word it a in way that lets users know that **if** they have changed any settings in `conf/settings.xml`, then either `MAVEN_HOME` or the `maven.home` system property needs to be set?

That initial comment has just been resolved: The section is now gone. 😄